.travis.yml

```diff
@@ -1,14 +1,14 @@
 language: scala
 scala:
-  - 2.13.5
+  - 2.13.6
 jdk:
   - openjdk8
   - openjdk11
 
 before_script:
   # Download sbt because Travis can't find it automatically :(
-  - mkdir -p $HOME/.sbt/launchers/1.5.0/
-  - curl -L -o $HOME/.sbt/launchers/1.5.0/sbt-launch.jar https://repo1.maven.org/maven2/org/scala-sbt/sbt-launch/1.5.0/sbt-launch-1.5.0.jar
+  - mkdir -p $HOME/.sbt/launchers/1.5.3/
+  - curl -L -o $HOME/.sbt/launchers/1.5.3/sbt-launch.jar https://repo1.maven.org/maven2/org/scala-sbt/sbt-launch/1.5.3/sbt-launch-1.5.3.jar
```

Comment:
I wish there were a way to not make this so brittle...

Reply:
The newer version of Travis (the .com one and not the .org) might fix this.